materials/module-1/worksheets/2-peer-to-peer-file-sharing-on-a-mesh.md

```diff
@@ -0,0 +1,80 @@
+Peer-to-peer file sharing on a mesh
+===================================
+
```

Comment:
Consider a shell reference (maybe its own page?) Would cull from https://swcarpentry.github.io/shell-novice/01-intro/ and https://swcarpentry.github.io/shell-novice/02-filedir/

Reply:
Why here?

Reply:
I agree, if you haven't looked at it by this point its to late :+1:  

i put together a list of command we use in the workshop [here](https://cryptpad.fr/pad/#/1/edit/l9PHRZqsDbWOwKK-DwprFA/+Wplznxf+H7ZF5J22PDKdkrj/)  and passed it on to @Shrinks99  see if he can build something 1/2 decent.. if not then on well.